lisa/src/main/java/it/unive/lisa/analysis/nonrelational/value/NonRelationalValueDomain.java

```diff
@@ -16,4 +18,46 @@
  */
 public interface NonRelationalValueDomain<T extends NonRelationalValueDomain<T>>
 		extends NonRelationalDomain<T, ValueExpression, ValueEnvironment<T>> {
+
+	@Override
+	public default ValueEnvironment<T> assume(ValueEnvironment<T> environment, ValueExpression expression,
+			ProgramPoint pp) throws SemanticException {
+		return environment;
+	}
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public default T glb(T other) throws SemanticException {
```

Comment:
This should be defined only once in `NonRelationalDomain` instead of having a duplicate definition in `NonRelationalHeapDomain` and `NonRelationalValueDomain`. Also, no default definition should be present: as for other lattice operations, default behavior is to be implemented in a `BaseNonRelationalValueDomain` instead of here.